### DISCUSSION TITLE
请问有适用mac的文件？

### DISCUSSION THREAD

**Discussion by Molly5on** [taurusxin/ncmdump#21] [Q&A] [answered]
请问有适用mac的文件吗～

**Comment by taurusxin** [comment, marked as answer]
有的呀，release里就能下载，开头是darwin的就是mac的。

**Comment by jeffreyshou7** [comment]
如何转成flac?谢谢

**Comment by PunkFleet** [comment]
mac版本打开出现错误：
<img width="289" height="280" alt="image" src="https://github.com/user-attachments/assets/e77384d1-58cf-43cb-b681-7cb8176614ec" />
